system_test/aest_oracles_SUITE.erl

```diff
@@ -0,0 +1,249 @@
+                62,238,132>>,
+    privkey => <<59,130,10,50,47,94,36,188,50,163,253,39,81,120,89,219,72,88,68,
+                 154,183,225,78,92,9,216,215,59,108,82,203,25,103,28,85,70,70,
+                 73,69,117,178,180,148,246,81,104,33,113,6,99,216,72,147,205,
+                 210,210,54,3,122,84,195,62,238,132>>
+}).
+
+-define(ALICE, #{
+    pubkey => <<177,181,119,188,211,39,203,57,229,94,108,2,107,214, 167,74,27,
+                53,222,108,6,80,196,174,81,239,171,117,158,65,91,102>>,
+    privkey => <<145,69,14,254,5,22,194,68,118,57,0,134,66,96,8,20,124,253,238,
+                 207,230,147,95,173,161,192,86,195,165,186,115,251,177,181,119,
+                 188,211,39,203,57,229,94,108,2,107,214,167,74,27,53,222,108,6,
+                 80,196,174,81,239,171,117,158,65,91,102>>
+}).
+
+-define(NODE1, #{
+    name    => node1,
+    peers   => [],
+    backend => aest_docker,
+    source  => {pull, "aeternity/epoch:local"}
+}).
+
+-define(NODE2, #{
+    name    => node2,
+    peers   => [node1],
+    backend => aest_docker,
+    source  => {pull, "aeternity/epoch:local"}
+}).
+
+%=== COMMON TEST FUNCTIONS =====================================================
+
+all() -> [
+    test_simple_same_node_query,
+    test_simple_two_nodes_query,
+    test_oracle_ttl_extension
+].
+
+init_per_suite(Config) ->
+    [
+        {node_startup_time, 20000}, %% Time may take to get the node to respond to http
+        {node_shutdown_time, 20000} %% Time it may take to stop node cleanly
+    | Config].
+
+init_per_testcase(_TC, Config) ->
+    aest_nodes:ct_setup(Config).
+
+end_per_testcase(_TC, Config) ->
+    aest_nodes:ct_cleanup(Config).
+
+end_per_suite(_Config) -> ok.
+
+%=== TEST CASES ================================================================
+
+test_simple_same_node_query(Cfg) ->
+    Opts = #{
+        oracle_node => node1,
+        oracle_id => ?OLIVIA,
+        querier_node => node1,
+        querier_id => ?ALICE
+    },
+    simple_query_test(Opts, Cfg).
+
+test_simple_two_nodes_query(Cfg) ->
+    Opts = #{
+        oracle_node => node1,
+        oracle_id => ?OLIVIA,
+        querier_node => node2,
+        querier_id => ?ALICE
+    },
+    simple_query_test(Opts, Cfg).
+
+simple_query_test(Opts, Cfg) ->
+    #{
+        oracle_node := ONode,
+        oracle_id := OAccount,
+        querier_node := QNode,
+        querier_id := QAccount
+    } = Opts,
+
+    MPubKey = maps:get(pubkey, ?MIKE),
+    OPubKey = maps:get(pubkey, OAccount),
+    QPubKey = maps:get(pubkey, QAccount),
+    EncMPubKey = aec_base58c:encode(account_pubkey, MPubKey),
+    EncOPubKey = aec_base58c:encode(oracle_pubkey, OPubKey),
+    EncQPubKey = aec_base58c:encode(account_pubkey, QPubKey),
+
+    %% Setup nodes
+    NodeConfig = #{ beneficiary => EncMPubKey },
+    setup([?NODE1, ?NODE2], NodeConfig, Cfg),
+    NodeNames = [node1, node2],
+    start_node(node1, Cfg),
+    start_node(node2, Cfg),
+    wait_for_startup([node1, node2], 4, Cfg),
```

Comment:
Note that this requires: 
system_test/aest_channels_SUITE_data/epoch.yaml.mustache:    beneficiary_reward_delay: 2

Reply:
which is indeed set in the file!